Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -1074,8 +1074,8 @@ def list_endpoint_info(account_id, client_id, auth_token, ip, hostname, USE_SSL)
                         'MACAddress': assets['nics'][0]['mac_address'],
                         'OS': assets['os_info']['os_platform'],
                         'OSVersion': assets['os_info']['os_version'],
-                        'Model': str(assets.get('computer_info', {}).get('model')),
-                        'Memory': assets['memory'],
+                        'Model': assets.get('computer_info', {}).get('model'),
+                        'Memory': assets.get('memory'),
                         'Assets': assets
```

Comment:
```suggestion
                        'Model': assets.get('computer_info', {}).get('model'),
                        'Memory': assets.get('memory', {}),
                        'Assets': assets
```

Reply:
Hey one more small modification please add this default value to the get method in case the memory returns empty it will return a default empty dict (The same type as the memory when returned from the assets)